packages/site/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/snap-simple-keyring-site",
-  "version": "1.1.6",
+  "version": "1.1.7",
```

Comment:
What are your thoughts on bumping the version to 2.0.0 rather than 1.1.7? Obviously there aren't any breaking changes in this release, but as it is 1.1.7 will be the initial version of the package, and since this is a special case where previously we were pushing changes to GitHub but now we are also pushing new versions to NPM, perhaps the initial version of the NPM package should be something a little less random.

``` suggestion
  "version": "2.0.0",